syntax = "proto3";

message NoiseMeasurement {
  uint32 room_id = 1;
  uint32 device_id = 2;
  float noise_value = 3;
}
